ui/lib/apps/ClusterInfo/components/StoreLocation.tsx

```diff
@@ -53,7 +53,12 @@ export default function StoreLocation() {
     <div>
       <ErrorBar errors={[error]} />
       <AnimatedSkeleton showSkeleton={isLoading}>
-        <StoreLocationTree dataSource={treeData} />
+        <StoreLocationTree
+          dataSource={treeData}
+          getMinHeight={
+            () => document.documentElement.clientHeight - 80 - 48 * 2 // 48 = margin of cardInner}
```

Comment:
The comment has a redundant `}`.

From ubyte: Because we listen to the resize event inside the StoreLocationTree, so we need to pass a function instead of a value to StoreLocationTree, else we need to listen to resize event as well in outside.